packages/fxa-auth-server/lib/routes/linked-accounts.ts

```diff
@@ -44,6 +45,29 @@ export class LinkedAccountHandler {
     }
   }
 
+  // As generated tokens expire after 6 months (180 days) per Apple documentation,
+  // generate JWT for client secret on each request instead
+  async generateAppleToken() {
+    const { clientId, keyId, privateKey, teamId } = this.config.appleAuthConfig;
+
+    if (!privateKey || !clientId || !keyId || !teamId) {
```

Comment:
I think we can pass these values into the function so that we wouldn't need to reference the config file and do config check earlier.